### PR TITLE
Add missing ansible configurations

### PR DESCRIPTION
**Pull request by NipunaPrashan**

### CODE REVIEW

File: roles/analytics-worker/tasks/main.yml [hunk 1]
```diff
@@ -58,10 +68,10 @@
...
       when: pack_location == "remote"
 
-#    - name: Copy JDBC Driver
-#      copy:
-#       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
-#       dest: "{{ carbon_home }}/repository/components/lib"
+    - name: Copy JDBC Driver to Analytics folder
+      copy:
+       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
+       dest: "{{ carbon_home }}/wso2/analytics/lib"
```
Comment (Savidude) [inline roles/analytics-worker/tasks/main.yml]: Add a condition to check if a JDBC driver has been defined.

Refer https://github.com/wso2/ansible-apim/blob/master/roles/apim/tasks/main.yml#L95

File: roles/analytics-dashboard/tasks/main.yml [hunk 2]
```diff
@@ -58,17 +69,32 @@
...
       when: pack_location == "remote"
 
-#    - name: Copy JDBC Driver
-#      copy:
-#       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
-#       dest: "{{ carbon_home }}/repository/components/lib"
+    - name: Copy JDBC Driver
+      copy:
+       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
+       dest: "{{ carbon_home }}/wso2/analytics/lib"
```
Comment (Savidude) [inline roles/analytics-dashboard/tasks/main.yml]: Check if JDBC driver is defined

File: files/misc/jartobundle.sh [hunk 3]
```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
```
Comment (Savidude) [inline files/misc/jartobundle.sh]: Shall we keep the misc directory empty in the GitHub repository?

File: roles/analytics-dashboard/tasks/custom.yml [hunk 4]
```diff
@@ -15,3 +15,7 @@
 #----------------------------------------------------------------------------
 
 # Add configs for the custom files that are needed to be copied/replaced here.
+ - name: "Copy jar to bundle script"
```
Comment (Savidude) [inline roles/analytics-dashboard/tasks/custom.yml]: Keep the custom files empty. This is to be used by users to add their own customizations.

File: roles/analytics-dashboard/tasks/main.yml [hunk 5]
```diff
@@ -58,17 +69,32 @@
...
+      }}/wso2/analytics/lib/{{jdbc_driver }} {{ carbon_home }}/wso2/analytics/lib"
+      environment:
+        JAVA_HOME: "{{java_home}}"
+      when: jdbc_driver is defined
+
+    - name: Remove default database connector jar file
+      file:
+        path: "{{ carbon_home }}/wso2/analytics/lib/{{jdbc_driver}}"
+        state: absent
+
```
Comment (Savidude) [inline roles/analytics-dashboard/tasks/main.yml]: Do not have custom scripts in the GitHub project.

File: roles/analytics-dashboard/tasks/main.yml [hunk 6]
```diff
@@ -83,9 +109,6 @@
     become_method: sudo
     become_user: root
 
-# Enable customization by uncommenting the following and adding the changes to the custom.yml
-      # - import_tasks: custom.yml
-
```
Comment (Savidude) [inline roles/analytics-dashboard/tasks/main.yml]: Keep this instruction

File: roles/analytics-worker/tasks/custom.yml [hunk 7]
```diff
@@ -15,3 +15,7 @@
 #----------------------------------------------------------------------------
 
 # Add configs for the custom files that are needed to be copied/replaced here.
+ - name: "Copy jar to bundle script"
+   template:
+     src: "{{ product_package_location }}/misc/jartobundle.sh"
+     dest: "{{ carbon_home }}/wso2/analytics/bin"
```
Comment (Savidude) [inline roles/analytics-worker/tasks/custom.yml]: Only include examples in the custom scripts.

File: roles/analytics-worker/tasks/main.yml [hunk 8]
```diff
@@ -78,6 +88,21 @@
...
+      }}/wso2/analytics/lib/{{jdbc_driver }} {{ carbon_home }}/wso2/analytics/lib"
+      environment:
+        JAVA_HOME: "{{java_home}}"
+      when: jdbc_driver is defined
+
+    - name: Remove default database connector jar file
+      file:
+        path: "{{ carbon_home }}/wso2/analytics/lib/{{jdbc_driver}}"
+        state: absent
+
```
Comment (Savidude) [inline roles/analytics-worker/tasks/main.yml]: Remove custom configurations

File: roles/integrator/templates/carbon-home/conf/carbon.xml.j2 [hunk 9]
```diff
@@ -655,8 +655,8 @@
             enable publishing events to the servers in a round robin manner.
             e.g.; tcp://analytics-server-1:7612,tcp://analytics-server-2:7612
          -->
-    <ServerURL>tcp://localhost:7612</ServerURL>
-    <AuthServerURL>ssl://localhost:7712</AuthServerURL>
+        <ServerURL>tcp://{{analytics_worker_hostname}}:7612</ServerURL>
+        <AuthServerURL>ssl://{{analytics_worker_hostname}}:7712</AuthServerURL>
```
Comment (Savidude) [inline roles/integrator/templates/carbon-home/conf/carbon.xml.j2]: Fix indentation

Comment (NipunaPrashan) [reply roles/integrator/templates/carbon-home/conf/carbon.xml.j2]: Fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-worker/tasks/main.yml (hunk 8 above)
Comment (NipunaPrashan) [reply roles/analytics-worker/tasks/main.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-worker/tasks/custom.yml (hunk 7 above)
Comment (NipunaPrashan) [reply roles/analytics-worker/tasks/custom.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-dashboard/tasks/main.yml (hunk 6 above)
Comment (NipunaPrashan) [reply roles/analytics-dashboard/tasks/main.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-dashboard/tasks/main.yml (hunk 5 above)
Comment (NipunaPrashan) [reply roles/analytics-dashboard/tasks/main.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-dashboard/tasks/custom.yml (hunk 4 above)
Comment (NipunaPrashan) [reply roles/analytics-dashboard/tasks/custom.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-dashboard/tasks/main.yml (hunk 2 above)
Comment (NipunaPrashan) [reply roles/analytics-dashboard/tasks/main.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

File: roles/analytics-worker/tasks/main.yml (hunk 1 above)
Comment (NipunaPrashan) [reply roles/analytics-worker/tasks/main.yml]: fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243

**Pull request merged**